### ISSUE TITLE
[server] ERROR: Could not invoke method

### ISSUE THREAD

**Issue by cyberboh** [Genymobile/scrcpy#3628]
- [x] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [x] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Windows
 - scrcpy version: [e.g. 1.24]
 - installation method: [Windows release...]
 - device model: Poco f3
 - Android version: [13 Dec SPL]

**Describe the bug**
I get this error as soon as i connect the device

On errors, please provide the output of the console (and `adb logcat` if relevant).

```
[server] ERROR: Could not invoke method
java.lang.NoSuchMethodException: android.content.IClipboard$Stub$Proxy.addPrimaryClipChangedListener [interface android.content.IOnPrimaryClipChangedListener, class java.lang.String, int]
        at java.lang.Class.getMethod(Class.java:2103)
        at java.lang.Class.getMethod(Class.java:1724)
        at com.genymobile.scrcpy.wrappers.ClipboardManager.getAddPrimaryClipChangedListener(ClipboardManager.java:103)
        at com.genymobile.scrcpy.wrappers.ClipboardManager.addPrimaryClipChangedListener(ClipboardManager.java:111)
        at com.genymobile.scrcpy.Device.<init>(Device.java:103)
        at com.genymobile.scrcpy.Server.scrcpy(Server.java:65)
        at com.genymobile.scrcpy.Server.main(Server.java:335)
        at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
        at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:360)
```

Please do not post screenshots of your terminal, just post the content as text instead.

**Comment by rom1v** [comment]
Thank you for your report.

This is a duplicate of #3497.

Specifically, please test: https://github.com/Genymobile/scrcpy/issues/3497#issuecomment-1359094255

**Comment by cyberboh** [comment]
Thank you, fixed in 1.25
